velox/expression/Expr.h

```diff
@@ -438,6 +442,9 @@ class ExprSet {
 
   std::vector<std::shared_ptr<Expr>> exprs_;
 
+  // Fields referenced by multiple expressions in ExprSet.
+  std::unordered_set<FieldReference * FOLLY_NONNULL> multiRefFields_;
```

Comment:
nit: its preferable to not use abbreviations, can you please rename this to multiplyReferencedFields_ here and above

Reply:
OK